activity/src/test/java/com/amazonaws/stepfunctions/cloudformation/activity/DeleteHandlerTest.java

```diff
@@ -26,28 +29,36 @@ public void setup() {
         request = ResourceHandlerRequest.<ResourceModel>builder()
                 .region(REGION)
                 .awsAccountId(AWS_ACCOUNT_ID)
-                .desiredResourceState(ResourceModel.builder().arn(ACTIVITY_ARN).name(ACTIVITY_NAME).build())
+                .desiredResourceState(ResourceModel.builder()
+                        .arn(ACTIVITY_ARN)
+                        .name(ACTIVITY_NAME)
+                        .build()
+                )
                 .build();
     }
 
     @Test
     public void testSuccess() {
```

Comment:
nit: while we're updating the test, maybe we can also update the test name to reflect the scenario under test to be more descriptive. The current name is vague